roles/openshift_logging/tasks/install_logging.yaml

```diff
@@ -1,8 +1,38 @@
 ---
+- debug:
+    msg: "openshift_logging_namespace var is deprecated. Ignoring inventory variable"
```

Comment:
this should be added to here instead: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_sanitize_inventory/vars/main.yml#L6

We get a lot of cool things letting `openshift_sanitize_inventory` check for deprecated vars like having them printed out at the end of the play instead of being buried in the role output like with `debug`